Mods/ArchipelagoPlayer.SC2Mod/enUS.SC2Data/LocalizedData/GameHotkeys.txt

```diff
@@ -587,3 +587,25 @@ Button/Hotkey/AP_BansheeMengskNightAssaultOff=T
 Button/Hotkey/AP_BansheeMengsk=E
 Button/Hotkey/AP_KerriganVoidCoopEconDrop=A
 Button/Hotkey/AP_KerriganVoidCoopCrushingGripWave=Z
+Button/Hotkey/AP_ExterminationProtocol=V
+Button/Hotkey/AP_SummonYggdrasil=O
+Button/Hotkey/AP_YggdrasilAssimilate=T
+Button/Hotkey/AP_YggdrasilEvolutionChamber=T    
+Button/Hotkey/AP_YggdrasilGreaterSpire=T
+Button/Hotkey/AP_YggdrasilHatchery=T
+Button/Hotkey/AP_YggdrasilHive=T
+Button/Hotkey/AP_YggdrasilHydraliskDen=T
+Button/Hotkey/AP_YggdrasilInfestationPit=T
+Button/Hotkey/AP_YggdrasilLair=T
+Button/Hotkey/AP_YggdrasilLurkerDen=T
+Button/Hotkey/AP_YggdrasilNydusNetwork=T
+Button/Hotkey/AP_YggdrasilPredatorNest=T
+Button/Hotkey/AP_YggdrasilRoachWarren=T
+Button/Hotkey/AP_YggdrasilSpawningPool=T
+Button/Hotkey/AP_YggdrasilSpineCrawler=T
+Button/Hotkey/AP_YggdrasilSpire=T
+Button/Hotkey/AP_YggdrasilSporeCrawler=T
+Button/Hotkey/AP_YggdrasilUltraliskCavern=T
+Button/Hotkey/AP_YggdrasilBanelingNest=T
+Button/Hotkey/AP_YggdrasilExtractor=T
+Button/Hotkey/AP_YggdrasilAutomatedExtractor=T
```

Comment:
Link the hotkeys together as they are in one slot and you want those rebindable in the hotkey editor (the build buttons should have hotkey entry of `Button/Hotkey/AP_YggdrasilAssimilate`)

Reply:
Should be fixed within the buttons section.